web/js/phoneNumberView.js

```diff
@@ -15,12 +15,18 @@
     Utils.addEventsHandlers('roomView:', eventHandlers, global);
   };
 
-  var init = function () {
-    LazyLoader.dependencyLoad([
-      '/js/vendor/jquery-1.11.1.min.js',
-      'https://cdnjs.cloudflare.com/ajax/libs/intl-tel-input/12.1.0/js/intlTelInput.js',
-      'https://cdnjs.cloudflare.com/ajax/libs/intl-tel-input/12.1.0/js/utils.js'
-    ]).then(function () {
+  var init = function (useGoogleApi) {
+    var dependencyLibs = [];
+
+    if(aParams.useGoogleApi)
+      dependencyLibs.push('//ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js');
+    else
+      dependencyLibs.push('//static.opentok.com/js/vendor/jquery-1.11.1.min.js');
```

Comment:
It's too bad we can't remove jQuery and use `document.querySelector()`. We only call `$()` once. But the intlTelInput library seems to require a jQuery object (not a DOM object). (With version 15.0 of the intlTelInput, this may be fixed. But I didn't look into it much.) 

Also, see my comment below about loading from a specific path at static.opentok.com.